release/macos/distribution.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8" standalone="no"?>
 <installer-script minSpecVersion="1.000000">
-    <title>alphaDIA 1.7.2</title>
+    <title>AlphaDIA 1.7.2</title>
```

Comment:
have the changes in the `distribution.xml` and the `info.plist` been tested?

Reply:
yup I tested all installers